tox.ini

```diff
@@ -76,7 +76,6 @@ commands =
 commands =
     make -f Makefile-docker update_deps
     pytest -m "static_assets" --ignore=tests/ui/ -v src/olympia/ {posargs}
-    make -f Makefile-docker update_assets
```

Comment:
The cool thing about this is that it allowed us to test the Makefile command, but I guess we can live without.

Reply:
We can always force it being used for our ui-tests later if needed or add explicit tests for the makefile commands. But I kinda thought having stable tests first is more important :laughing: